pkg/api/postage_test.go

```diff
@@ -232,6 +232,48 @@ func TestPostageGetStamps(t *testing.T) {
 	})
 }
 
+func TestPostageGetNonExistingStamps(t *testing.T) {
+	ts, _, _, _ := newTestServer(t, testServerOptions{DebugAPI: true, Post: nil, BatchStore: nil, BlockTime: nil})
```

Comment:
here you need to pass in a `mockpost`  and a nil batchstore so that the stamp issuer returns an issuer but no batch is returned from the batchstore.